saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -447,14 +462,23 @@ class OrderRefundProductsInput(graphene.InputObjectType):
         graphene.NonNull(OrderRefundFulfillmentLineInput),
         description="List of fulfilled lines to refund.",
     )
+    payments_to_refund = graphene.List(
+        PaymentToRefundInput,
+        required=False,
+        description=f"{ADDED_IN_31} Payments that need to be refunded.",
+    )
     amount_to_refund = PositiveDecimal(
         required=False,
-        description="The total amount of refund when the value is provided manually.",
+        description=(
+            "The total amount of refund when the value is provided manually. "
+            f"{DEPRECATED_IN_3X_INPUT} Use paymentsToRefund instead."
+        ),
     )
     include_shipping_costs = graphene.Boolean(
         description=(
-            "If true, Saleor will refund shipping costs. If amountToRefund is provided"
-            "includeShippingCosts will be ignored."
+            "If true, Saleor will refund shipping costs. If amountToRefund is provided "
+            "includeShippingCosts will be ignored. "
+            f"{DEPRECATED_IN_3X_INPUT} Use paymentsToRefund instead."
```

Comment:
@korycins Shouldn't we keep it where it is? Dashboard designs have only one "include_shipping_costs" checkbox. It hard or me to imagine the logic of handling it per payment.

Reply:
Hhm... Yes, you're right. We don't need it, we ignore `includeShippingCosts` when `amountToRefund` is provided, so we could just add it to the total refund amount, but not sure how we should handle the case, when part of payments to refund has provided a refund amount, and the rest of the payments doesn't have it.  (last point from this comment:https://github.com/mirumee/saleor/pull/8165#discussion_r716547245)